hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpGetCommandProcessor.java

```diff
@@ -117,7 +117,7 @@ private void handleHealthReady(HttpGetCommand command) {
         Node node = textCommandService.getNode();
 
         if (node.isRunning()
-                && node.getNodeExtension().isStartCompleted()) {
```

Comment:
`isStartCompleted()` doing the same thing, why did you create `isReady()` method?

Reply:
it is overridden on EE side to support properly `/health/ready` when persistence is enabled